catalog/opensearch/catalog-opensearch-source/src/test/java/ddf/catalog/source/opensearch/impl/TestOpenSearchSource.java

```diff
@@ -110,6 +114,69 @@
 
     private static FilterBuilder filterBuilder = new GeotoolsFilterBuilder();
 
+    private static InputStream getSampleAtomStreamWithForeignMarkup() {
+        String response =
```

Comment:
I'm definitely not opposed to that; I was just doing it this way to keep it consistent with what is already being done in this test class.